vendor/experimental-render-mode-rehydrate.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  if (typeof FastBoot === 'undefined') {
+    var current = document.getElementById('fastboot-body-start');
+
+    if (current && current.nextSibling.nodeValue === '%+b:0%') {
+      Ember.ApplicationInstance.reopen({
```

Comment:
Is this how we plan to do this long term? Do we have a plan to remove this reopen?